Cache/Redis.php

```diff
@@ -3,7 +3,8 @@
 namespace Smartive\HandlebarsBundle\Cache;
 
 use Handlebars\Cache;
-use Predis\Client as Predis;
+use Predis\Client as PredisClient;
+use Redis as RedisClient;
```

Comment:
yes, its just a naming alias. php is not java, even if it now has namespaces :-P

https://3v4l.org/iYNQV